src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person: {
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+} }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+    {age && (<p className="Person__age">{`I am ${age}`}</p>)}
+    <p className="Person__partner">
+      {isMarried ? `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}` : 'I am not married'}
```

Comment:
Great! but don't use ternary operator inside the ternary operator)
this is too much.
use condition to know which text you should render.
```
isMarried
  ? isPartner
  : 'I am not married'
```